centreon/packages/ui/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,2 @@
+// TODO type IconProps = { name: 'add' | '...' };
+// TODO <Icon name="icon-name" />
```

Comment:
as the `/components/*` are generally WIP / in flux, it can contain some todo / notes for future direction
that said, if you prefer to keep these type of todo notes out of the code base, then I'll manage them in a separate doc